content/news/buzz/december-2024-status-update/contents.lr

```diff
@@ -0,0 +1,66 @@
+title: December 2024 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+Work in December (and the publication of this update!) was slowed by the end-of-year break; but BeeWare still made some significant progress to round out the year.
+
+What we've done
+----------------
+
+* We added a `GTK implementation of Geolocation services <https://github.com/beeware/toga/pull/2999>`__.
+
+* We made a number of improvements to Pack, Toga's style API. This includes `adding a justify-config style directive <https://github.com/beeware/toga/pull/3054>`__, `adding a gap style directive <https://github.com/beeware/toga/pull/3036>`__, and `updating a number of Pack properties to match their CSS names <https://github.com/beeware/toga/pull/3033>`__.
+
+* We `standardised the order in which Toga widgets are instantiated <https://github.com/beeware/toga/pull/2942>`__. This simplifies the implemenation process, and guaranteeing that the initial styles added to a widget are always applied.
+
+* We added an API for `reporting and controlling the state of Toga windows <https://github.com/beeware/toga/pull/2473>`__.
+
+* We modified Toga's web backend to be `compatible with Micropython <https://github.com/beeware/toga/pull/2976>`__. Micropython is significantly smaller that the full Pyodide install, reducing the startup time for web apps.
```

Comment:
So far this work hasn't touched the backend, only the core, and even there it isn't complete:

```suggestion
* We began modifying Toga to be `compatible with MicroPython <https://github.com/beeware/toga/pull/2976>`__. MicroPython is significantly smaller than the full CPython install, reducing the startup time for web apps.
```